docs/content/users/quickstart.md

```diff
@@ -576,17 +582,17 @@ Importing database...
 Successfully imported database for drupal8
 ```
 
-#### Supported file types
+#### Supported File Types
 
-Database import supports the following file types:
+Database imports can be any of the following file types:
 
 * Raw SQL Dump (.sql)
 * Gzipped SQL Dump (.sql.gz)
```

Comment:
```suggestion
* Gzipped SQL Dump (`.sql.gz`)
* Xz'd SQL Dump (`.sql.xz`)
```

Reply:
Bah, good catch! Will try and remember to add backticks for the other file extensions in this list if I missed those too.

Reply:
`Xz'd` will likely not make the cut with the spellchecker (or you) :)

Reply:
I’ll admit it’s a verb I’ve never used.